packages/itinerary-body/src/styled.tsx

```diff
@@ -269,7 +269,7 @@ export const ItineraryBody = styled.ol`
 export const LegBody = styled.div`
   color: #676767;
   font-size: 13px;
-  padding: 12px 0 12px 4px;
+  padding-bottom: 12px;
 `;
 
 export const LegClickable = styled.div``;
```

Comment:
While we're in here, thoughts on centering the content here? I think it looks a lot cleaner:
```suggestion
export const LegClickable = styled.div`
  height: 60px;
  display: flex;
  align-items: center;
`;
```

Reply:
It works nicely with the transit legs, but less good with walk legs. Maybe something for the full team to discuss!